drafts/repl.it/introduction-to-repl-it-ide.md

```diff
@@ -91,7 +91,7 @@ def solve_quadratic(a, b, c):
     d = (b ** 2) -4 * a * c
     s1 = (-b+math.sqrt((b**2)-(4*(a*c))))/(2*a)
```

Comment:
I would recommend changing the code to use the variable `d`.
Also adding spacing will also make it more readable. 
`s1 = (-b+math.sqrt((b**2)-(4*(a*c))))/(2*a)` -> `s1 = (-b + math.sqrt(d)) / (2 * a)`